components/slices/NavigationZone.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="navigation-zone">
+    <div v-for="(menuItem, index) in navigationLinks" :key="`item-${index}`">
+      <pix-link
+        v-if="!menuItem.hasOwnProperty('subNavigationLinks')"
+        :field="menuItem.link"
+        class="navigation-zone__item"
+      >
+        {{ $prismic.asText(menuItem.name) }}
+      </pix-link>
+      <div v-else>
+        <button
+          :dropdown-index="`${index}`"
+          class="dropdown-toggle navigation-zone__item links-group"
+          @click="toggleDropdown(`${index}`)"
+        >
+          {{ menuItem.name }}
+          <fa v-if="showDropdown(`${index}`)" icon="angle-up" />
+          <fa v-else icon="angle-down" />
+        </button>
+        <navigation-dropdown
+          v-if="showDropdown(`${index}`)"
+          type="button"
+          :options="menuItem.subNavigationLinks"
+          :dropdown-index="`${index}`"
+          @closeNavigationDropdown="toggleDropdown(`${index}`)"
+        >
+        </navigation-dropdown>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import NavigationDropdown from '@/components/NavigationDropdown'
+
+class Navigation {
+  constructor() {
+    this.links = []
+  }
+
+  addNavigationLink(navigationLink) {
+    this.links.push(navigationLink)
+  }
+
+  addSubNavigationLink(groupName, subNavigationLink) {
```

Comment:
Est-ce que c'est bien une méthode locale ?
```suggestion
  _addSubNavigationLink(groupName, subNavigationLink) {
```

Reply:
Non, c'est une méthode publique